src/custom-elements/src/constants/constants.ts

```diff
@@ -50,7 +50,7 @@ export type DateFormats = typeof DATE_FORMATS[number];
 export const CHI_TIME_AUTO_SCROLL_DELAY = 50;
 export const CHI_TIME_SCROLL_ADJUSTMENT = 24;
 
-export const TIME_PICKER_FORMATS = ['12hours', '24hours'];
+export const TIME_PICKER_FORMATS = ['12hr', '24hr'] as const;
```

Comment:
add `as const`, so type `TimePickerFormats` is `"12hr" | "24hr"` now 

without it type of `TimePickerFormats` is `string`